wgsl/Makefile

```diff
@@ -8,8 +8,9 @@ clean:
 	rm -f index.html index.pre.html grammar/grammar.js wgsl.lalr.txt
 
 # Generate spec HTML from Bikeshed source.
-index.pre.html: index.bs wgsl.recursive.bs.include
-	DIE_ON=everything bash ../tools/invoke-bikeshed.sh index.pre.html index.bs wgsl.recursive.bs.include
+WGSL_SOURCES=index.bs wgsl.recursive.bs.include wgsl.reserved.bs.include
```

Comment:
Should we do *.bs.include if that works to ensure future PRs' includes just work with previews? (I was authoring almost the same change, that's where my include PR came from, haha)

Reply:
Sure, that's a good idea.

Reply:
Thank you!

Reply:
I made it "wgsl.*.bs.include", an used Gnu Makefile syntax for expanding the wildcard.